test/samples/async.js

```diff
@@ -19,7 +19,7 @@ module.exports = [
 	{
 		description: 'errors on async function declarations if transform is enabled',
 		input: `async function A() {}`,
-		error: /Transforming async functions is not implemented/
+		output: `function A() { return Promise.resolve(); }`
```

Comment:
I think you can leave the `error` and add `output`. That way, the tests still pass until we remove the `error` property.